pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.test</groupId>
+    <artifactId>device</artifactId>
+    <version>0.1.0</version>
```

Comment:
You should be using a maven SNAPSHOT version during development, not a release one.

Reply:
```suggestion
    <version>0.1.0-SNAPSHOT</version>
```
Would it be OK?